VehicleUI/src/main/java/com/drivequant/drivekit/vehicle/ui/vehicles/utils/VehicleUtils.kt

```diff
@@ -86,48 +83,21 @@ object VehicleUtils {
     fun getVehicleDrawable(context: Context, vehicleId: String): Drawable? {
         val defaultVehicleDrawable = ResourcesCompat.getDrawable(context.resources, getFilterVehicleDrawable(vehicleId), null)
```

Comment:
Pour éviter de créer le drawable dans tous les cas, on peut peut-être transformer en fonction ?
```suggestion
        val defaultVehicleDrawable = {
            ResourcesCompat.getDrawable(context.resources, getFilterVehicleDrawable(vehicleId), null)
        }
```
Et du coup appeler `defaultVehicleDrawable()` dans les 2 `return`

Reply:
Plutôt que de créer une fonction sans paramètre d'entrée, on pourrait plutôt utiliser ici un `by lazy` sur `defaultVehicleDrawable` ?